lib/modules/dosomething/dosomething_campaign/dosomething_campaign.helpers.inc

```diff
@@ -84,7 +84,7 @@
  *   - vips (string). HTML.
  *
  */
-function dosomething_campaign_load($node, $public = FALSE) {
+function dosomething_campaign_load($node, $public = FALSE, $language = DOSOMETHING_GLOBAL_DEFAULT_LANG_CODE) {
```

Comment:
does this default value actually work? it's not being included as a constant at the top of the file